GPU/TPCFastTransformation/macro/TPCFastTransformInit.C

```diff
@@ -21,19 +21,24 @@
 /// root -l TPCFastTransformInit.C'("debugVoxRes.root")'
 ///
 
+#include "Algorithm/RangeTokenizer.h"
+
```

Comment:
```suggestion
```
Already below

Reply:
@wiechula , 

the macro does not work for me without this header.  

the interpreter does not understand this line:
 auto y2xBins = o2::RangeTokenizer::tokenize<float>(userInfo->FindObject("y2xBinning")->GetTitle());

Reply:
@sgorbuno , yes, but now it is included twice. I think the macro should be used compiled, then the `#if ...` should include it. I never had problems with it.